### DISCUSSION TITLE
All blockings get lost when Mullvad browser is closed

### DISCUSSION THREAD

**Discussion by Fritz66** [uBlockOrigin/uAssets#21296] [Q&A] [answered]
All blocks get lost when Mullvad browser is closed

Every time I close my browser, all my personal blocks are lost. Is there any way to fix this?

Version: 1.54.0

**Comment by ghost** [comment]
Personal blocks = ?

![image](https://github.com/uBlockOrigin/uAssets/assets/152656980/2b9798ee-fc22-4094-b7c8-da6d87f0b5a7)

**Reply by ghost** [reply]
Or this

![padlock](https://user-images.githubusercontent.com/95879668/172276144-f3be783c-b0c1-45d2-b926-88d793e3fde4.png)

**Reply by Fritz66** [reply]
Yes, it's on personal blocks

**Comment by Fritz66** [comment]
> Personal blocks = ?
> 
> ![image](https://private-user-images.githubusercontent.com/152656980/289270910-2b9798ee-fc22-4094-b7c8-da6d87f0b5a7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDMwNzE3NTMsIm5iZiI6MTcwMzA3MTQ1MywicGF0aCI6Ii8xNTI2NTY5ODAvMjg5MjcwOTEwLTJiOTc5OGVlLWZjMjItNDA5NC1iN2M4LWRhNmQ4N2YwYjVhNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIyMFQxMTI0MTNaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNmY0N2ZhYTNlMThhMjIyNzgwNDE1ZmUwZjJkMTNmY2ZmNGY2MDQxNGQ2NDkwMzNhZmFiM2M5ZTUzYTU0YzEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jGRAbPr4l2IRzLxmQN6E4gdUwthBUKJzUDtcui5FPO8)



> Or this
> 
> ![padlock](https://user-images.githubusercontent.com/95879668/172276144-f3be783c-b0c1-45d2-b926-88d793e3fde4.png)

I try to block ugly avatars in forums. As far as I remember this wasn't an issue before, it just worked.

**Reply by ghost** [reply]
Is "My filters" selected in "Filter lists" tab in uBO dashboard?

**Comment by Fritz66** [comment, marked as answer]
Guys, I finally fixed it.
I removed µBlock Origin, which was installed in two different directories, and reinstalled it. Now everything is fine, my personal blocks are still available after closing the browser :)
